freqtrade/commands/arguments.py

```diff
@@ -116,8 +116,15 @@
 
 NO_CONF_ALLOWED = ["create-userdir", "list-exchanges", "new-strategy"]
 
-ARGS_STRATEGY_UTILS = ["strategy_list", "strategy_path", "recursive_strategy_search"]
+ARGS_STRATEGY_UPDATER = ["strategy_list"]
```

Comment:
i do like the variable name better - but `"strategy_path", "recursive_strategy_search"` should be left where they are ... 
using a different strategy path (or recursive search) is a quite common usecase...

Reply:
done